otter/test/test_controller.py

```diff
@@ -1510,19 +1505,32 @@ class ModifyAndTriggerTests(SynchronousTestCase):
     """
 
     def setUp(self):
-        self.group = util_mock_group("state", "t", "g")
+        self.state = sample_group_state()
+        self.group = util_mock_group(self.state, 'tid', 'gid')
         self.addCleanup(set_config_data, {})
         self.mock_tg = patch(self, "otter.controller.trigger_convergence",
                              side_effect=intent_func("tg"))
         self.logargs = {"a": "b"}
         self.disp = SequenceDispatcher([
             (BoundFields(mock.ANY, self.logargs),
-             nested_sequence([(("tg", "t", "g"), noop)]))
+             nested_sequence([(("tg", "tid", "gid"), noop)]))
         ])
 
     def modify(self, group, state):
         return "newstate"
 
+    def test_tenant_suspended(self):
+        """
+        Raises :obj:`TenantSuspendedError` if associated tenant is suspended.
```

Comment:
I prefer to say "fails with" rather than "raises", since `delete_group` doesn't raise anything synchronously.